swift/Tooling/Templates/Workflow (Verbose).xctemplate/___FILEBASENAME___Workflow.swift

```diff
@@ -79,8 +79,8 @@ extension ___VARIABLE_productName___Workflow {
 
 extension ___VARIABLE_productName___Workflow {
 
-    func compose(state: ___VARIABLE_productName___Workflow.State, context: WorkflowContext<___VARIABLE_productName___Workflow>) -> Screen {
-        #warning("Don't forget your compose implementation!")
+    func compose(state: ___VARIABLE_productName___Workflow.State, context: WorkflowContext<___VARIABLE_productName___Workflow>) -> ___VARIABLE_productName___Screen {
```

Comment:
This makes it our our template can’t compile out of the box. Which to me is a bummer because I often find myself “filling out” the template from top to bottom. Could we use a type that actually exists?

Reply:
I agree that it's a bummer. On the first use of a template, there won't be any type that exists that we can use (aside from AnyScreen, but absolutely don't want it to be that).

My other thought was to define a type with a comment as a placeholder. I *don't* want to encourage definition of the screen type to be done in the workflow.

Maybe something like:
```
// Delete this type and replace the return type of `compose` with the actual screen this workflow will output
struct PlaceholderScreen {}

func compose(state: ___VARIABLE_productName___Workflow.State, context: WorkflowContext<___VARIABLE_productName___Workflow>) -> PlaceholderScreen {
```

What do you think?

Reply:
Updated this after talking with @bencochran, there are lots of types that we could return from compose.

Having it default to a String, so that it compiles immediately when the template is created.